microservice-occasion/README.md

```diff
@@ -69,8 +73,8 @@ Before causing a fallback, ensure that you followed the [Getting Started](../REA
 
```

Comment:
Just above  I see "mvn liberty:stop-server"  is that different when using gradle?

Reply:
done.